find_xml_file.rb

```diff
@@ -0,0 +1,15 @@
+
+def find_xml_file(folder)
+  files = Dir.entries(folder)
+
+  xml_file = nil
+
+  files.each do |filename|
+    if filename.match "\.xml$"
+      xml_file = filename
+      break
+    end
+  end
```

Comment:
This will only return the first xml file in the provided folder. Consider the case that multiple .xml files might exist.